nexus_constructor/instrument_view.py

```diff
@@ -60,16 +70,25 @@ def __init__(self, parent):
         }
 
         for _ in range(self.num_neutrons):
-            self.neutron_objects["entities"].append(Qt3DCore.QEntity(self.root_entity))
+            self.neutron_objects["entities"].append(
+                Qt3DCore.QEntity(self.component_root_entity)
+            )
             self.neutron_objects["meshes"].append(Qt3DExtras.QSphereMesh())
             self.neutron_objects["transforms"].append(Qt3DCore.QTransform())
 
         self.cylinder_length = 40
```

Comment:
this is ok to leave undocumented in my opinion, and isn't really a magic number, because the variable name describes what its purpose is.